viewing_party/main.py

```diff
@@ -0,0 +1,100 @@
+# Nandita Gilroy's Viewing Party Project!
+
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title and genre and rating:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    else:
+        new_movie = None
+    return new_movie
```

Comment:
You can clean this up a little using an inline dictionary:

```suggestion
    new_movie = None
    if movie_title and genre and rating:
        new_movie = {
            "title": movie_title,
            "genre": genre,
            "rating": rating
        }
    return new_movie
```

Reply:
Alternately:

```suggestion
    if movie_title and genre and rating:
        return = {
            "title": movie_title,
            "genre": genre,
            "rating": rating
        }
    else:
        return None
```